app/instance-initializers/head.js

```diff
@@ -1,30 +1,33 @@
 import ENV from '../config/environment';
 
 export function initialize(owner) {
-  if (ENV['ember-cli-head'] && ENV['ember-cli-head']['suppressBrowserRender']) { return true; }
+  if (typeof FastBoot === 'undefined') {
+    if (ENV['ember-cli-head'] && ENV['ember-cli-head']['suppressBrowserRender']) { return true; }
 
-  // clear fast booted head (if any)
-  let startMeta = document.querySelector('meta[name="ember-cli-head-start"]')
-  let endMeta = document.querySelector('meta[name="ember-cli-head-end"]')
-  if (startMeta && endMeta) {
-    let el = startMeta.nextSibling
-    while(el && el !== endMeta) {
-      document.head.removeChild(el);
-      el = startMeta.nextSibling;
+    // clear fast booted head (if any)
+    let startMeta = document.querySelector('meta[name="ember-cli-head-start"]')
+    let endMeta = document.querySelector('meta[name="ember-cli-head-end"]')
+    if (startMeta && endMeta) {
+      let el = startMeta.nextSibling
+      while(el && el !== endMeta) {
+        document.head.removeChild(el);
+        el = startMeta.nextSibling;
+      }
+      document.head.removeChild(startMeta);
+      document.head.removeChild(endMeta);
     }
-    document.head.removeChild(startMeta);
-    document.head.removeChild(endMeta);
-  }
 
-  const component = owner.lookup('component:head-layout');
-  component.appendTo(document.head);
+    const component = owner.lookup('component:head-layout');
+    component.appendTo(document.head);
+  } else {
```

Comment:
You are merging the fastboot initializer here and sending the extra bytes to browser that will never be run. Ideally you can move the new existing `app/instance-initializer/fastboot/head.js` to `fastboot/instance-initializers/head.js`. That's how we envisioned apps moving. Also currently the rc builds auto migrate the fastboot initializers to their new location with throwing a warning asking addons to the do the same. 

I would suggest you to take that approach and then this addon does a major version bump.